cypress/integration/features/teams/createEditDeleteTeam.feature

```diff
@@ -0,0 +1,41 @@
+Feature: CRUD operation on Teams
+
+  As a teacher I want to create/edit/delete the team so that I can manage the team.
+
+Scenario: Creating a new team by Teacher
+  Given I am logged in as a 'teacher' at 'brb'
+  When I go to teams overview
+  When I click on add team button
```

Comment:
"When I click on add team button"
We don't have defined how to write the feature file steps, right?
I usually write the name of buttons in capitals -> When I click on button Add Team
Should we find a rule how to handle this?

Reply:
discussed and updated.